### PR TITLE
Enhance Readme

### PR DESCRIPTION
**Pull request by rubo77**
This will reorder the steps, so you can work in another screen configuring and preparing everything while the slow build bṕrocess runs.

### CODE REVIEW

**Comment by rubo77** [comment]
I rebased this to the latest commits.

This PR also includes some typos and minor anhancements in the Readme

**Comment by RalfJung** [comment]
The build process takes <10min even on our not-very-fast server... I do not think it is worth reordering everything for that. After all, the initial setup will only happen once.

Either people already know how to use screen, they can just do that without further hand-holding. Or they don't, then I do not think this is the right place to teach them.

**Comment by rubo77** [comment]
But you should only clean up if everything is working fine, before that, you could have to rebuild some times

**Comment by RalfJung** [comment]
Why should you have to rebuild? Something is seriously wrong if the build fails... we have CI to make sure that does not happen.

**Comment by rubo77** [comment]
On my server it took about an hour the first time. depends on your server i guess. I wanted to work during that time and get it all done fast, so I just changed the order of putting up the systemd service with the nginx config.

the cleaning is optional and the end is a good place I think.

I split those up in two commits

**Comment by RalfJung** [comment]
> I split those up in two commits

I don't think I'd like the README to be complicated by the parallel build instructions. That just doesn't seem worth it. However, I'd like to take the improvements from your first two commits. So could you remove the last commit?

File: README.md [hunk 1]
```diff
@@ -68,38 +70,38 @@ the steps accordingly.
...
+7.  We will use the `ff-node-monitor` system user to access PostgreSQL, and we
     need to create a database for the service:
 
     ```
     sudo -u postgres psql -c 'CREATE ROLE "ff-node-monitor" WITH LOGIN;'
     sudo -u postgres psql -c 'CREATE DATABASE "ff-node-monitor" WITH OWNER = "ff-node-monitor" LC_COLLATE = '\''de_DE.utf8'\'' TEMPLATE template0;'
     ```
 
-    You may have to install the `de_DE.UTF-8` locale before this works.  On
+    You may have to install the `de_DE.UTF-8` locale before this works. On
```
Comment (RalfJung) [inline README.md]: I am consistently using two spaces after a full stop. Please keep that style.

File: README.md [hunk 2]
```diff
@@ -14,6 +14,8 @@ I have tested the following steps on a Debian Stretch system; if you are using a
 different version or a different distribution, you might have to change some of
 the steps accordingly.
 
+Make sure, you have at least 1.5 GB of free diskspace.
```
Comment (RalfJung) [inline README.md]: Spurious comma, please remove.

File: README.md (hunk 1 above)
Comment (rubo77) [reply README.md]: why did you choose this style with two spaces? It is not known to me, that this is correct. (Seems weird)

Comment (rubo77) [reply README.md]: >  So could you remove the last commit?

done

Comment (RalfJung) [reply README.md]: > But why did you choose this style with two spaces? It is not known to me, that this is correct. (Seems weird)

It is what emacs uses when you let it auto-wrap text. It matches the good typesetting practice to make the space after a full stop bigger than the space between words, increasing readability (any good typesetting system will do that). Usually the space shouldn't be twice as big, but there's only so much you can do with a fixed-width font.

File: README.md [hunk 3]
```diff
@@ -112,10 +114,12 @@ the steps accordingly.
     If the last command does not show the service as running, you need to debug
     and fix whatever issue has come up.
 
-3.  To expose the service on the internet, set up a reverse proxy in your main
-    web server.  Here's how that could look like for nginx (this is a snippet of
+10. To expose the service on the internet, set up a reverse proxy in your main
+    web server. Here's how that could look like for `nginx` (this is a snippet of
```
Comment (RalfJung) [inline README.md]:
```suggestion
    web server.  Here's how that could look like for `nginx` (this is a snippet of
```

**Comment by RalfJung** [comment]
Thanks! I'll fix the remaining nits after merging.

**Pull request merged**